### DISCUSSION TITLE
Encountered a connection error

### DISCUSSION THREAD

**Discussion by HJCM-2** [the-anylogic-company/Alpyne#21] [Q&A] [answered]
Hello, I am a beginner who has just started learning Anylogic and Alpyne. I am trying to run the simplest "simple run - single.py" mentioned in the video: https://www.youtube.com/watch?v=v-fx4CGZ9aE. 

When I run in Mac, I encounter “ConnectionRefusedError: [Errno 61] Connection refused“ error, use Windows will also encounter ”ConnectionRefusedError: [WinError 10061] No connection could be made because the target machine actively refused it.“ May I ask where is the problem? Thank you very much for your help :)

**Comment by HJCM-2** [comment, marked as answer]
Java >= 1.11, put **examples_and_tests.zip** from **releases** into **Alpyne/alpyne** folder.

**Reply by kuma1064** [reply]
I have encountered the same Problem as u. Can u explain a bit more detailed what exactly you did to fix it?
Many thanks in advance.

**Reply by HJCM-2** [reply]
Sorry for the late answer. You must make sure your java version >1.11. Download **examples_and_tests.zip** from the end of **https://github.com/t-wolfeadam/Alpyne/releases/tag/v0.2.0**. Unzip and put it in your downloaded  main code **Alpyne/alpyne** folder. Use the newly downloaded example folder.
